src/app/_components/bridge-now-link.tsx

```diff
@@ -15,14 +17,24 @@ export function BridgeNowLink(props: Props) {
   );
 }
 
-function _BridgeNowLink({ className, ...props }: Props) {
+function _BridgeNowLink({ className, section, ...props }: Props) {
   const searchParams = useSearchParams();
 
   const refParams = searchParams.get("ref") || searchParams.get("referrer");
   const bridgeNowLink = `${bridgeAppBaseUrl}/bridge${refParams ? `?ref=${refParams}` : ""}`;
 
+  const pathname = usePathname();
+  const actionCallback = useCallback(() => {
```

Comment:
Is this memoization really needed?

Reply:
Fair - I've removed it in place of a lambda function.